src/msgpack.light/MsgPackContext.cs

```diff
@@ -48,19 +50,20 @@ public class MsgPackContext
 
         private readonly Dictionary<Type, Func<object>> _objectActivators = new Dictionary<Type, Func<object>>();
 
-        public MsgPackContext()
+        public void Initialize()
```

Comment:
This is an antipattern. You should give a detailed explanation, why do we need it.